CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 
 ## [Unreleased 5.1.11]
+## Features
+- AVAILABILITY REQUEST : add See Sale Order Line button
```

Comment:
changelog message should be 'AVAILABILITY REQUEST : Addition of button on grid-view to view sale order lines.